functions/go/apply-setters/README.md

```diff
@@ -7,9 +7,9 @@
 Apply setter values on resource fields. Setters serve as parameters for template-free
 setting of field values.
 
-Setters are a safer alternative to other substitution techniques which do not
-have the context of the structured data. Setters may be invoked to modify the
-package resources using this function to set desired values.
+Setters are a safer alternative to parameterize field values as they
+have the context of the structured data. Setters can be invoked to modify the
+resources using this function to set desired values.
```

Comment:
How about other fishfood feedback? One notable one is to define the concepts and terminology precisely. What are the different nouns:
- setter key
- setter value
- setter reference comment

etc.

Reply:
Added the definitions.